pkgs/applications/networking/seaweedfs/default.nix

```diff
@@ -1,22 +1,22 @@
 { lib
 , fetchFromGitHub
-, buildGoModule
+, buildGo118Module
```

Comment:
Please move this to top-level so it can easily be cleaned with the go 1.18 bump.